### PR TITLE
chore(manifest): add cdn static support for existing S3

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/manifest/env.go [hunk 1]
```diff
@@ -270,7 +271,19 @@ func (cfg *EnvironmentCDNConfig) UnmarshalYAML(value *yaml.Node) error {
 	return nil
 }
 
-// IsEmpty returns true if vpc is not configured.
+// CDNStaticConfig represents the static config for CDN.
+type CDNStaticConfig struct {
+	Location string  `yaml:"location,omitempty"`
```
Comment (Lou1415926) [inline internal/pkg/manifest/env.go:276]: Does this work nicely with environment overrides `mergo`?

**Review by Lou1415926 — COMMENTED** [review]
LGTM just one questeion!

File: internal/pkg/manifest/env.go (hunk 1 above)
Comment (iamhopaul123) [reply internal/pkg/manifest/env.go:276]: This is an env manifest field. How do we use env overrides 🤔

Comment (Lou1415926) [reply internal/pkg/manifest/env.go:276]: :homer-disappearing

**Review by Lou1415926 — APPROVED** [review]
:shipit:

File: internal/pkg/manifest/env.go [hunk 2]
```diff
@@ -214,8 +214,9 @@ type EnvironmentCDNConfig struct {
 
 // AdvancedCDNConfig represents an advanced configuration for a Content Delivery Network.
 type AdvancedCDNConfig struct {
-	Certificate  *string `yaml:"certificate,omitempty"`
-	TerminateTLS *bool   `yaml:"terminate_tls,omitempty"`
+	Certificate  *string         `yaml:"certificate,omitempty"`
+	TerminateTLS *bool           `yaml:"terminate_tls,omitempty"`
+	Static       CDNStaticConfig `yaml:"static,omitempty"`
```
Comment (efekarakus) [inline internal/pkg/manifest/env.go]: What do you think of the following for the key name?
```suggestion
	Static       CDNStaticConfig `yaml:"static_website,omitempty"`
```
It's slightly more verbose but I think more explicit 🤔

**Review by efekarakus — APPROVED** [review]
Looks great, approved with do-not-merge to hear your thoughts on the field name

File: internal/pkg/manifest/env.go (hunk 2 above)
Comment (iamhopaul123) [reply internal/pkg/manifest/env.go]: It's actually where their static assets go. I wonder if `static_website` can correctly convey the meaning 🤔

Comment (efekarakus) [reply internal/pkg/manifest/env.go]: That totally makes sense! How about `static_content` or `static_assets` then?

**Pull request merged**